tests/NA12878_1000genomes-dragen-3.7.6/output_certified/results/multiqc/multiqc_data/multiqc_data.json

```diff
@@ -2,16 +2,16 @@
     "report_data_sources": {
         "Picard": {
             "InsertSizeMetrics": {
-                "NA12878": "/home/ubuntu/2023/NPM-sample-qc/tests/NA12878_1000genomes-dragen-3.7.6/work/76/892ac114b1a8a8627f32f3f92510d9/NA12878.insert_size_metrics.txt"
+                "NA12878": "/home/ubuntu/2023/reads_mapped/NPM-sample-qc/tests/NA12878_1000genomes-dragen-3.7.6/work/c9/c9b5caf20c181a30025f970e6b1c26/NA12878.insert_size_metrics.txt"
```

Comment:
I noticed that the `output_certified` folder has mentioned the path of temporary files.
Do we care about displaying a "clean" path ?
current commit: `home/ubuntu/2023/reads_mapped/NPM-sample-qc/tests/`
instead of ~cleaner: `/home/ubuntu/NPM-sample-qc/tests/`

Reply:
yes it is not very pretty ... but let's not care about this